app/assets/javascripts/fae/navigation/_navigation.js

```diff
@@ -207,85 +205,6 @@ Fae.navigation = {
   },
 
   /**
-   * Click event of the admin search in the main nav
-   */
-  utilitySearch: function() {
-    var $search_wrapper = $('.utility-search-wrapper');
-
-    $('.js-utility-dropdown').hover(function() {
-      $search_wrapper.hide();
-    });
-
-    $('#js-utility-search').click(function() {
-      $search_wrapper.toggle();
-
-      if($search_wrapper.is(':visible')) {
-        $search_wrapper.find('input').focus();
-      }
-
-    });
-  },
-
-  /**
-   * Initialize the typeahead navigation in the header nav
-   */
-  typeaheadSearch: function() {
```

Comment:
If you're scrubbing Typeahead, could you delete the files in `vendor/javascripts` too?